### PR TITLE
Remove redundant value in shorthand properties

### PR DESCRIPTION
**Pull request by SoffiMeower**
### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
'0 2em 2em 2em' -> '0 2em 2em'
'20px 0 0 0' -> '20px 0 0'
'14px 14px' -> '14px'
'0 auto 1em auto' -> '0 auto 1em'
'5em 2em 0 2em' -> '5em 2em 0'

### CODE REVIEW

File: website/client/assets/scss/form.scss [hunk 1]
```diff
@@ -85,8 +85,8 @@ input, textarea, input.form-control, textarea.form-control {
     &.input-group-icon {
       border: solid 1px $gray-400;
       border-right: none;
-      border-top-right-radius: 0px;
-      border-bottom-right-radius: 0px;
+      border-top-right-radius: 0;
+      border-bottom-right-radius: 0;
```
Comment (LikeLakers2) [inline website/client/assets/scss/form.scss]: Forgive me if I'm missing something, but doesn't this sort of change seem inconsistent with all the other times that a single value is used in this file?

edit: Nevermind, I'm surprised I didn't notice that a lot of the other instances of a `0` that already exist, they don't have an appropriate identifier (i.e. `px`) after it. Feel free to ignore this.

**Comment by paglias** [comment]
Fixed some conflicts generated by changes in the client side code

**Comment by paglias** [comment]
Thanks for the PR @ghostslayer989 and sorry for the delay. After some thinking I think that we prefer to keep the values in CSS as explicit as possible, in particular we prefer not to change `5em 2em 0 2em` to `5em 2em 0` as it makes the code less clear and the file size savings are neglegible.

 If you want to remove those changes and keep only the other ones we'll be happy to accept the PR

**Comment by SabreCat** [comment]
Closing due to the length of time awaiting response to review comments.

**Pull request closed without merging**